activiti-cloud-runtime-bundle-service/activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-impl/src/test/java/org/activiti/cloud/services/rest/controllers/ProcessInstanceControllerImplIT.java

```diff
@@ -201,6 +201,18 @@ public void createProcess() throws Exception {
             .andDo(document(DOCUMENTATION_IDENTIFIER + "/create"));
     }
 
+    @Test
+    public void startCreatedProcess() throws Exception {
+        StartProcessPayload cmd = ProcessPayloadBuilder.start().withProcessDefinitionId("1").build();
+        when(processRuntime.startCreatedProcess("1")).thenReturn(defaultProcessInstance());
+
+        mockMvc.perform(post("/v1/process-instances/{processInstanceId}/start", 1)
+            .contentType(APPLICATION_JSON)
+            .content(mapper.writeValueAsString(cmd)))
```

Comment:
Actually, I think this is wrong. This endpoint does not need any payload, right? it's only a post on the process instance id.